plugin/trino-hive/src/main/java/io/trino/plugin/hive/CachingDirectoryLister.java

```diff
@@ -179,4 +233,27 @@ public long getRequestCount()
     {
         return cache.stats().requestCount();
     }
+
+    private static String createCacheKey(String databaseName, String tableName, String path)
+    {
+        return format("%s###%s###%s", databaseName, tableName, path);
```

Comment:
Any suggestions on how to get to more elegant keys based on the  `databaseName`, `tableName` and `path` ?